ManageWikiSettings.php

```diff
@@ -175,21 +175,6 @@
 		'section' => 'anti-spam',
 		'help' => 'Auto-review settings for edits/new pages.',
 	],
-	'wmgFlaggedRevsRestrictionLevels' => [
-		'name' => 'FlaggedRevs Restriction Levels',
-		'from' => 'flaggedrevs',
-		'restricted' => false,
-		'type' => 'usergroups',
```

Comment:
You can change this to userrights

Reply:
No we can't. Per @JohnFLewis it bloats the page and severally slows down ManageWikiSettings.